salt/cloud/deploy/bootstrap-salt.sh

```diff
@@ -227,7 +227,7 @@ _SALT_MASTER_ADDRESS=${BS_SALT_MASTER_ADDRESS:-null}
 _SALT_MINION_ID="null"
 # _SIMPLIFY_VERSION is mostly used in Solaris based distributions
 _SIMPLIFY_VERSION=$BS_TRUE
-_LIBCLOUD_MIN_VERSION="0.14.0"
+_LIBCLOUD_MIN_VERSION="2.0.0"
```

Comment:
I honestly don't know who updates the the repos. But I could certainly help. I'll look at salt-bootstrap. Something is causing the version errors